ycms/cms/templates/zentral.html

```diff
@@ -0,0 +1,86 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% block content %}
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "YCMS at a glance" %}</div>
+            <div class="grid grid-cols-3">
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Wards" %}</div>
+                    <div class="text-3xl font-bold">{{ wardscount }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Available Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ availablebeds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Doctors" %}</div>
+                    <div class="text-3xl font-bold">{{ doctors }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ beds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Occupied Beds" %}</div>
+                    <div class="text-3xl font-bold">{{ occupiedbeds }}</div>
+                </div>
+                <div>
+                    <div class="text-lg font-semibold">{% translate "Total Nurses" %}</div>
+                    <div class="text-3xl font-bold">{{ nurses }}</div>
+                </div>
+                <div class="col-span-2 sm:col-span-1">
+                    <div class="text-lg font-semibold">{% translate "Ward Details" %}</div>
+                    <div class="text-2xl">
+                        <ul>
+                            {% for ward in wards %}<li>{{ ward }}</li>{% endfor %}
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <br>
+    <div class="col-span-1 sm:col-span-2 md:col-span-2 lg:col-span-2 xl:col-span-2">
+        <div class="rounded-lg border border-gray-300 bg-white p-4 shadow-md flex flex-col h-full">
+            <div class="mb-4 text-2xl font-semibold">{% translate "Add a New Ward" %}</div>
+            <br>
+            <form enctype="multipart/form-data"
+                  method="post"
+                  id="ward_form"
+                  class="mb-12"
+                  data-unsaved-warning>
+                {% csrf_token %}
+                {{ wardform }}
+                <br>
+                <button type="submit" class="btn mt-4">{% translate "Add New Ward" %}</button>
+            </form>
```

Comment:
```suggestion
            <form enctype="multipart/form-data"
                  method="post"
                  id="ward_form"
                  class="mb-12 flex justify-between items-center"
                  data-unsaved-warning>
                {% csrf_token %}
                {% for field in wardform %}
                    <div>
                        <label for="{{ field.id_for_label }}"
                               {% if field.field.required %}class="field-required"{% endif %}>{{ field.label }}</label>
                        {% render_field field %}
                        <div class="help-text">{{ field.help_text }}</div>
                    </div>
                {% endfor %}
                <button type="submit" class="btn mt-4">{% translate "Add New Ward" %}</button>
            </form>
```

I know this is more verbose, but IMO looks way nicer since this is a small form:
![image](https://github.com/charludo/ycms/assets/47758554/0099fcd1-4ee1-4157-823e-f589cab78b41)